### ISSUE TITLE
Accessibility: ensure all URLs on Home Page are distinct from regular text

### ISSUE THREAD

**Issue by hujambo-dunia** [galaxyproject/galaxy-hub#2434]
@hexylena, writes:
the direct links in the 'publications', 'events' and 'tutorials' cards fail WCAG 1.4.1, but more than that, it is not clear at all that the events are clickable links.
They should be underlined or otherwise distinguished, but with the presence of the 'visit more' (probably should be 'view more') button it is confusing that they are not just text. Some examples for what passes/fails: https://www.tpgi.com/well-color-us-surprised-this-sc-can-be-a-tricky-customer/ I would recommend a :hover effect as well.

**Comment by hujambo-dunia** [comment]
As of Feb 22, 2024 the Home Page was giving an 0/zero "serious"/"critical" errors as per WCAG AAA 2.2 at least per [this Chrome extension tool](https://accessibleweb.com/). Nevertheless, @hexylena's comment makes sense and is worth looking into.
